frameworks/cassandra/src/main/dist/svc.yml

```diff
@@ -313,7 +534,35 @@ pods:
                     cqlsh -u cassandra -p cassandra {{{CASSANDRA_CQLSH_SSL_FLAGS}}} -e "alter keyspace system_traces      WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 3};" "node-0-server.${FRAMEWORK_HOST}" {{TASKCFG_ALL_CASSANDRA_NATIVE_TRANSPORT_PORT}} &&
                     cqlsh -u cassandra -p cassandra {{{CASSANDRA_CQLSH_SSL_FLAGS}}} -e "alter keyspace system_auth        WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 3};" "node-0-server.${FRAMEWORK_HOST}" {{TASKCFG_ALL_CASSANDRA_NATIVE_TRANSPORT_PORT}} &&
                     cqlsh -u cassandra -p cassandra {{{CASSANDRA_CQLSH_SSL_FLAGS}}} -e "alter keyspace system_distributed WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 3};" "node-0-server.${FRAMEWORK_HOST}" {{TASKCFG_ALL_CASSANDRA_NATIVE_TRANSPORT_PORT}} &&
-                    nodetool repair system_auth &&
+                    {{#TASKCFG_ALL_SECURE_JMX_ENABLED}}
+                    ./bootstrap --resolve=false
+                    cd $MESOS_SANDBOX
+                    export HOME=$MESOS_SANDBOX
+                    export SECURE_JMX_KEY_STORE_KEY=`cat jmx/key_store_pass`
+                    export KEY_STORE_PATH=jmx/key_store
+                    if [ -f "jmx/trust_store" ]; then
+                        # load user provided trust store
+                        export TRUST_STORE_PATH=jmx/trust_store
+                        export SECURE_JMX_TRUST_STORE_KEY=`cat jmx/trust_store_pass`
+                    else
+                        # load user provided keystore certs to default truststore
+                        export TRUST_STORE_PATH=$JAVA_HOME/lib/security/cacerts
+                        export SECURE_JMX_TRUST_STORE_KEY="changeit"
+                        ${JAVA_HOME}/bin/keytool -noprompt -importkeystore -srckeystore $KEY_STORE_PATH -srcstorepass $SECURE_JMX_KEY_STORE_KEY -destkeystore $TRUST_STORE_PATH -deststorepass $SECURE_JMX_TRUST_STORE_KEY
+                    fi
+
+                    mkdir jmx_properties
+                    mkdir .cassandra
+                    (umask u=rw,g=,o= && cp jmx/key_file jmx_properties/key_file)
+                    chmod +x .ssl/jmx-ssl-properties.sh
+                    .ssl/jmx-ssl-properties.sh
+                    export HOME=$MESOS_SANDBOX &&
```

Comment:
You already exported home [above](https://github.com/mesosphere/dcos-commons/pull/3146/files#diff-fe0c20da1fba698643bc101afa99c204R540)